openstudiocore/src/utilities/core/FileReference.cpp

```diff
@@ -150,7 +150,7 @@ void FileReference::setPath(const openstudio::path& newPath) {
 bool FileReference::makePathAbsolute(const openstudio::path& searchDirectory) {
```

Comment:
I hope by deprecated you mean we can eliminate it. I think we'll just have to see how tangled it really is.